data/cpu_bugs/vm_install_script/sle-15/create_vm_with_exist_disk.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -e
+
+
+if [ $# -lt 4 ]; then
+	echo "[Error]: too few arguments" >&2
+	echo "[Notice]: You should make sure the <VM_NAME>-<SUBNAME>.qcow2 at <VM_POOL>" >&2
+	echo "Usage:"
+	echo "$0 <VM_NAME> <SUBNAME> <VM_POOL> <CPU_FEATRURE>" >&2
+	exit 1;
+fi
+
+VM_NAME=$1
```

Comment:
Before you used parameter expansion, why not here ?

Reply:
:-), OK.
No problem. I can use that way again.

Reply:
First time with expansion always with expansion :laughing:

Reply:
There is a similar culture in China. :-)